From 0a4c5db225d8d213916655593471e303ba71e0ea Mon Sep 17 00:00:00 2001
From: Oleksandr Kravchuk <sashko@users.noreply.github.com>
Date: Mon, 13 Dec 2021 03:42:44 +0100
Subject: [PATCH] Include typeinfo for typeid() (#83)

Otherwise fails with:
    error: must '#include <typeinfo>' before using 'typeid'

Fetch from: https://github.com/dbus-cxx/dbus-cxx/pull/83
Upstream-Status: Accepted
Signed-off-by: Daniel Lang <d.lang@abatec.at>

---
 dbus-cxx/demangle.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dbus-cxx/demangle.h b/dbus-cxx/demangle.h
index b71bcb9..9a4e99b 100644
--- a/dbus-cxx/demangle.h
+++ b/dbus-cxx/demangle.h
@@ -21,6 +21,7 @@
 #define DBUSCXX_DEMANGLE_H
 
 #include <string>
+#include <typeinfo>
 #include <dbus-cxx/dbus-cxx-config.h>
 
 #if DBUS_CXX_HAS_CXXABI_H

